x-pack/plugins/features/server/oss_features.ts

```diff
@@ -37,6 +37,7 @@ export const buildOSSFeatures = ({
       privileges: {
         all: {
           app: ['discover', 'kibana'],
+          api: ['fileUpload:analyzeFile'],
```

Comment:
I know we discussed Discover as one example of a feature that should grant this functionality, but is that the only one? If I'm reading this right, it looks like we are taking this away from the `maps` and `ml` features, which would surprise our users when they upgrade

Reply:
Yes, failing tests have just shown that ML still needs to supply this privilege.
Maps does not as it does not use the file structure finder feature.

Reply:
Updated in https://github.com/elastic/kibana/pull/101169/commits/3d3c2c003243cf34f8063e6e2d297d343f4c3f21